docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -1,8 +1,8 @@
 # Install OpenVINO™ Runtime on Linux from an Archive File {#openvino_docs_install_guides_installing_openvino_from_archive_linux}
 
-With the OpenVINO™ 2022.2 release, you can download and use archive files to install OpenVINO Runtime. 
+With the OpenVINO™ 2022.2 release, you can download and use archive files to install OpenVINO Runtime. The archive files contain pre-built binaries and library files needed for OpenVINO Runtime, as well as sample code for running demos. This page provides instructions showing how to install OpenVINO Runtime using archive files. Check the Release Notes for more information on updates in the 2022.2 release.
```

Comment:
why do we remove the link to Release Notes here?

Reply:
Oops! There should still be a link there. I'll go back and add the link.